src/main/java/next/dto/UserCreatedDto.java

```diff
@@ -31,4 +31,14 @@ public String getName() {
     public String getEmail() {
         return email;
     }
+
+    @Override
+    public String toString() {
+        return "UserCreatedDto{" +
```

Comment:
이런 toString에서는 `ReflectionToStringBuilder`의 toString() 메서드가 유용해요.
[Java Docs - ReflectionToStringBuilder](http://commons.apache.org/proper/commons-lang/javadocs/api-3.9/org/apache/commons/lang3/builder/ReflectionToStringBuilder.html#toString-java.lang.Object-org.apache.commons.lang3.builder.ToStringStyle-)

저는 주로 아래와 같이 사용하고 있습니다.
```suggestion
        return ReflectionToStringBuilder.toString(this, ToStringStyle.DEFAULT_STYLE);
```